apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -229,6 +229,7 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     if (currentCwd !== buildFolder) {
       // Update the CWD to the project's build root. Some tools, like Jest, use process.cwd()
       this._terminal.writeVerboseLine(`CWD is "${currentCwd}". Normalizing to project build folder.`);
+      process.chdir('src');
```

Comment:
Why does changing the CWD to something else and then back to the buildfolder fix the issue?

What if the `src` folder doesn't exist?

Reply:
We verified process.chdir(process.cwd().toLowerCase()) doesn't do anything.
Not sure the potential reason, but it looks like if the only thing different is the casing, we'll have to bounce to a different directory first. 
Something like:
PS D:\rushstack> node -e "console.log(process.cwd())"
D:\rushstack
PS D:\rushstack> node -e "process.chdir('d:\rushstack'), console.log(process.cwd())"
D:\rushstack
PS D:\rushstack> node -e "process.chdir('D:\'), process.chdir('d:\rushstack'), console.log(process.cwd())"
d:\rushstack

Reply:
I'm looking for a folder that must exist in a project. Does such folder exist?

Reply:
Perhaps we can ensure the `.heft` folder has been created at this point and use that? That's the only one we can actually guarantee exists because it's the only one we can definitively control.

Reply:
Can we ensure that or we haven't? If not, we need to ensure that before doing chdir